### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by SolarDrew**
Fix some typos and a few hyperlinks.

### CODE REVIEW

File: docs/guide/acquiring_data/jsoc.rst [hunk 1]
```diff
@@ -161,16 +161,16 @@ If 2 or more PrimeKeys need to be passed together::
...
 
     >>> a.jsoc.Time('2014-01-01T00:00:00', '2014-01-01T01:00:00'), a.jsoc.PrimeKey('WAVELNTH', '161')  # doctest: +SKIP
 
 If the correct keyword is not specified, or the passed PrimeKey is not supported by the given series, a
-meaningful error will be thrown, which will give you the PrimeKey supported by that series. Hence, by looking
+meaningful error will be thrown, which will give you the PrimeKeys supported by that series. Hence, by looking
 at the error, one can easily retry building the query with correct PrimeKeys.
 
-Other important thing to note is that, Wavelength when passed through in-built attribute, should be passed as a
+Anther important thing to note is that, Wavelength when passed through in-built attribute, should be passed as an
```
Comment (Cadair) [inline docs/guide/acquiring_data/jsoc.rst]: Another

File: docs/guide/acquiring_data/jsoc.rst [hunk 2]
```diff
@@ -471,18 +471,18 @@ Making a query
 
 Querying JSOC using the JSOC client is very similar to what we were doing with Fido.
 As above, we have to make sure we have an email address registered with JSOC before you are allowed to make a request.
-See `this <http://jsoc.stanford.edu/ajax/register_email.html>_` to register your email address.
-We can add email address to the search query with the :mod:`jsoc.Notify` attribute.
+See `this <http://jsoc.stanford.edu/ajax/register_email.html>`_ to register your email address.
```
Comment (Cadair) [inline docs/guide/acquiring_data/jsoc.rst]: double `__`

File: sunpy/net/jsoc/attrs.py [hunk 3]
```diff
@@ -19,7 +19,7 @@ class Series(_VSOSimpleAttr):
     """
     The JSOC Series to Download.
 
-    This is the list of `Series <http://jsoc.stanford.edu/JsocSeries_DataProducts_map.html>_`.
+    This is the list of `Series <http://jsoc.stanford.edu/JsocSeries_DataProducts_map.html>`_.
```
Comment (Cadair) [inline sunpy/net/jsoc/attrs.py]: and here

File: sunpy/net/jsoc/jsoc.py [hunk 4]
```diff
@@ -99,7 +99,7 @@ class JSOCClient(object):
     .. warning::
 
         JSOC requires you to register your email address before requesting
-        data. `See this on how to register <http://jsoc.stanford.edu/ajax/register_email.html>_`.
+        data. `See this on how to register <http://jsoc.stanford.edu/ajax/register_email.html>`_.
```
Comment (Cadair) [inline sunpy/net/jsoc/jsoc.py]: and here

**Comment by SolarDrew** [comment]
Clearly didn't realise the double `_` thing, thanks @Cadair

**Comment by nabobalis** [comment]
Are you sure its double `_`?

**Comment by Cadair** [comment]
using double underscore for external links is good practice, as you can only use single once per link iirc.

**Pull request merged**